src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs

```diff
@@ -100,10 +100,18 @@ public RuntimeBroker(
                 s_lazyCore.Value.EnablePii(_logger.PiiLoggingEnabled);
             }
 
-            _parentHandle = GetParentWindow(uiParent);
+            if (DesktopOsHelper.IsWindows())
+            {
+                _parentHandle = GetParentWindow(uiParent);
+            }
+            else
+            {
+                // TODO:ADO 3055958 Parent window handle support on mac
```

Comment:
nit: Just out of curiosity, how will the broker window be handled on mac? (add a comment)

Reply:
I updated in the comment. Previously I did some investigation, but don't have a clear solution yet... I have a card about this topic and will do more research in the next quarter.